### PR TITLE
chore(graphql): upgrade rxdart 0.28.0

### LINKED ISSUE #1441: Update rxdart dependency
Currently the graphql package prevents us from upgrading to rxdart to 0.28.0.

### PR DESCRIPTION
**Pull request by ronnnnn**
Fixes #1441

Upgrade rxdart to 0.28.0.
Newer version might not affect any changes to this package.

- [rxdart changelog | Dart package](https://pub.dev/packages/rxdart/changelog)

### CODE REVIEW

**Review by vincenzopalazzo — DISMISSED** [review]
Thanks!

File: packages/graphql/pubspec.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ dependencies:
   collection: ^1.15.0
   web_socket_channel: '>=2.3.0 <=2.4.0'
   stream_channel: ^2.1.0
-  rxdart: ^0.27.1
+  rxdart: ^0.28.0
```
Comment (renefloor) [inline packages/graphql/pubspec.yaml:23]: I would go for 
```suggestion
  rxdart: '>=0.27.1 <0.29.0'
```
As you don't have a reason to drop 0.27.1, but I would love to see this merged anyway.

**Comment by vincenzopalazzo** [comment]
Commit message was wrong, please pay attention next time. See the correct way https://github.com/zino-hofmann/graphql-flutter/commit/1b851eaa9c64ce4bbf6f0962cd64a74209d95ce5

**Comment by renefloor** [comment]
@vincenzopalazzo, I didn't add any commit message. When you do 'apply suggestion' you set the commit message.

Any chance this gets published? The last published version is from 3 months ago. We cannot upgrade Flutter without updating rxdart due to some dependency conflicts.

**Comment by vincenzopalazzo** [comment]
>I didn't add any commit message. When you do 'apply suggestion' you set the commit message.

For this reason, I should not be the one changing the commit message

>Any chance this gets published? The last published version is from 3 months ago. We cannot upgrade Flutter without updating rxdart due to some dependency conflicts.

Lets do it now

**Pull request merged**